keystore/encrypt.go

```diff
@@ -13,6 +13,8 @@ import (
 	"path/filepath"
 
 	"github.com/ChainSafe/gossamer/crypto"
+	"github.com/ChainSafe/gossamer/crypto/ed25519"
+	"github.com/ChainSafe/gossamer/crypto/sr25519"
```

Comment:
Should we include secp in keystore encrypt?

Reply:
added